docs/usage/esql.asciidoc

```diff
@@ -0,0 +1,78 @@
+[[esql]]
+=== ES|QL in the Java client
+
+This page enables you to understand and use {ref}/esql.html[ES|QL] in the
+Java client.
```

Comment:
Should this be {java-client} instead? Not sure -- it's kind of nice to not have it exactly the same way every time :)